src/Compiler/ReleaseNotesBuilder.cs

```diff
@@ -85,19 +81,41 @@ Milestone GetPreviousMilestone()
                 .FirstOrDefault();
         }
 
-        string GetCommitsLink(Milestone previousMilestone)
+        string GetCommitsLink(Milestone targetMilestone, Milestone previousMilestone)
         {
             if (previousMilestone == null)
             {
-                return string.Format("https://github.com/{0}/{1}/commits/{2}", user, repository, targetMilestone.Title);
+                return $"https://github.com/{user}/{repository}/commits/{targetMilestone.Title}";
             }
-            return string.Format("https://github.com/{0}/{1}/compare/{2}...{3}", user, repository, previousMilestone.Title, targetMilestone.Title);
+            return $"https://github.com/{user}/{repository}/compare/{previousMilestone.Title}...{targetMilestone.Title}";
         }
 
-        void AddIssues(StringBuilder stringBuilder, List<Issue> issues)
+        void AddIssues(StringBuilder builder, List<Issue> issues)
         {
-            Append(issues, "Feature", stringBuilder);
-            Append(issues, "Bug", stringBuilder);
+            var bugs = issues
+               .Where(issue => issue.Labels.Any(label => label.Name == "Type: Bug"))
```

Comment:
Isn't that to wide, what if there is a label that contains those characters?

How about: 

`issue => issue.Labels.Any(label => label.Name == "Type: Bug" || label.Name == "Bug"`